packages/editor/src/components/commands/index.js

```diff
@@ -299,9 +300,10 @@ function useEditedEntityContextualCommands() {
 		};
 	}, [] );
 	const { openModal } = useDispatch( interfaceStore );
+	const hasEditorContentOverlay = useHasEditorContentOverlay();
 	const commands = [];
 
-	if ( postType === PATTERN_POST_TYPE ) {
+	if ( ! hasEditorContentOverlay && postType === PATTERN_POST_TYPE ) {
```

Comment:
When editing a page, there's an Edit Template command as well:

<img width="483" alt="Screenshot 2024-06-20 at 12 51 00 PM" src="https://github.com/WordPress/gutenberg/assets/6458278/5a1408ac-bde6-4db4-99bb-11d91a82ac7c">

Do you think that should also be hidden?

Reply:
I'm not sure I understand the change here to be honest?

Reply:
Yeah, might need a comment. 

When editing a pattern, and then activating the style book, this condition ensures the default pattern commands aren't shown by default when the command centre is opened:

<img width="448" alt="Screenshot 2024-06-20 at 4 39 53 PM" src="https://github.com/WordPress/gutenberg/assets/6458278/73f187bb-1d00-4e75-9bd7-5d87f3c2a227">

Reply:
Is this the right fix? Maybe when going into "style book" and "style revision", we should be changing the "context" instead. (See `useSetCommandContext`) Again, I think this is just a side effect of us not actually changing the active "entity" in the EditorProvider to the global styles entity, but that is definitely a bigger change.

Reply:
Good points! I've updated to have the `edit-site` packages remove the command context temporarily when Style Book and/or Style Revisions are open, clearing the suggested commands that are specific to editing the current entity (for both patterns and page): https://github.com/WordPress/gutenberg/pull/62669/files#diff-caaab4735e981969da0fb0108be1c6a34e2f039940615a695fa05e061dac42d9R40

> Again, I think this is just a side effect of us not actually changing the active "entity" in the EditorProvider to the global styles entity, but that is definitely a bigger change.

Agreed, I think this is another example supporting how opening a global styles entity would make more sense. But hopefully this workaround is acceptable for now, until we've made that much bigger change.